src/Pages/suyeonkim/Login/Login.js

```diff
@@ -1 +1,39 @@
-//
+import React from 'react';
+import LoginForm from './LoginForm/LoginForm';
+import ForgotPassword from './ForgotPassword/ForgotPassword';
+import './Login.scss';
+
+class Login extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      id: '',
+      pw: '',
+    };
+  }
+
+  handleIdInput = e => {
+    const inputTarget = e.target;
+
+    this.setState({
+      [inputTarget.name]: inputTarget.value,
+    });
+  };
```

Comment:
```suggestion
  handleIdInput = e => {
    const { name, value } = e.target;

    this.setState({ [name]: value });
  };
```
구조분해 할당 문법을 사용하면 불필요한 변수 선언을 줄일 수 있습니다!